File: Etherlendo.API/Etherlendo.API/Models/Project.cs
using ElasticSearchLite.NetCore.Interfaces;

namespace Etherlendo.API.Models
{
    public class Project : IElasticPoco
    {
        public string Id { get; set; }
        public string Type { get; set; }
        public string Index { get; set; }
        public double? Score { get; set; }
        public long Total { get; set; }
        public long Version { get; set; }

        public string Name { get; set; }
        public string Description { get; set; }
        public string ContractAddress { get; set; }
        public string Image { get; set; }
    }
}

File: Etherlendo.Web/src/app/project/models/project.ts
import { ProjectState } from "./project-state";

export class Project {
    id: string;
    name: string;
    description: string;
    image: string;
    started?: boolean;
    state?: ProjectState;
    contractAddress?: string;
    total?: number;
    interest?: number;
    fundingEndsAt?: Date;
    investedAmount?: number;
    investedPercentage?: number;
}